packages/core/src/styling/theme.ts

```diff
@@ -73,10 +74,19 @@ export type OldTheme = {
 }
 
 export type Theme = {
-  default?: true
-  defaultDark?: true
-  defaultLight?: true
+  default?: string
+  defaultDark?: string
+  defaultLight?: string
   propertyDefinitions?: Record<CustomPropertyName, CustomPropertyDefinition>
+  themes?: {
+    [key: string]: Record<
+      string,
+      {
+        order?: number
+      }
+    >
+  }
```

Comment:
I suppose this will be a map of available themes. I'm not sure I understand the nested structure here?

```suggestion
  themes?: Record<
      string,
      {
        order?: number
      }
    >
```